src/components/Lists/ListBase/ListBase.stories.tsx

```diff
@@ -30,22 +42,54 @@ const simpleListItems = [
 ];
 
 export const SimpleList: Story<ListBaseProps> = (args) => (
-    <ListBase {...args}>
-        <ListItemBase>
-            <ListItemIconBase color="secondary">
-                <InboxIcon />
-            </ListItemIconBase>
-            <ListItemTextBase primary="Inbox" color="secondary-main" />
-        </ListItemBase>
-
-        {simpleListItems.map((item, index) => (
-            <ListItemBase key={index}>
-                <ListItemIconBase>{item.icon}</ListItemIconBase>
-                <ListItemTextBase primary={item.text} />
+    <BoxBase width={360}>
+        <ListBase {...args}>
+            <ListItemBase button highlighted>
+                <ListItemIconBase>
+                    <InboxIcon />
+                </ListItemIconBase>
+                <ListItemTextBase primary="Inbox" />
             </ListItemBase>
-        ))}
-    </ListBase>
+
+            {simpleListItems.map((item, index) => (
+                <ListItemBase key={index} button>
+                    <ListItemIconBase>{item.icon}</ListItemIconBase>
+                    <ListItemTextBase primary={item.text} />
+                </ListItemBase>
+            ))}
+        </ListBase>

```

Comment:
I think highlighted should be in the simpleListItems's data

Reply:
That's a good idea! ListBase stories is very clean now. I don't know why I don't think about this earlier !!